TheraLang.BLL/Services/NewsService.cs

```diff
@@ -130,6 +132,25 @@ public async Task UpdateNews(int id, NewsEditDto newsDto)
             await _unitOfWork.SaveChangesAsync();
         }
 
+        public async Task Like(int id, User user)
```

Comment:
We got many-to-many relation (user - news), it woud be optimal to update like/dislike using junction table, though its not critical